gulpfile.js

```diff
@@ -47,19 +47,19 @@ function lintGulpfile ()
 
 function plugins (done)
 {
-    let pluginConfig = Object.create(webpackConf).slice(1);
+    var pluginConfig = Object.create(webpackConf).slice(1);
     webpack(pluginConfig).run(done);
 }
 
 function diva (done)
 {
-	let divaConfig = Object.create(webpackConf)[0];
+	var divaConfig = Object.create(webpackConf)[0];
     webpack(divaConfig).run(done);
 }
 
 function server ()
 {
-    let devConfig = Object.create(webpackConf)[0];
+    var devConfig = Object.create(webpackConf)[0];
```

Comment:
What was the error in jslint for these?

Reply:
This was (if I remember correctly) an error with strict mode and not allowing `let` to be used